behavioral-questions.md

```diff
@@ -4,16 +4,16 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: during highschool i remember the school starting to transition into a tech base school where they integrated coding, and data science classes. i was fortunate enough to try both and found that coding was harder but i enjoyed it more than analyzing data.
```

Comment:
One thing to always keep in mind when answering these questions is how an interviewer that is asking it may interpret or take the information that you are sharing.  For that reason, it can be helpful to spin these answers into interests you have in software development that would make you a good addition to their team.